UITests/Responses/TransferMethod/ListTransferMethodResponse.json

```diff
@@ -42,7 +42,7 @@
         },
         {
             "token": "trm-11111111-0000-0000-0000-000000000000",
-            "type": "BANK_ACCOUNT",
+            "type": "VENMO_ACCOUNT",

```

Comment:
This json doesn't match Venmo account response, Venmo account does not have bank id, branch id, bankAccountId etc fields